sympy/tensor/index_methods.py

```diff
@@ -24,6 +23,7 @@ class IndexConformanceException(Exception):
     pass
 
 
+@deprecated(useinstead='EinsteinSum', issue=9284)
 def _remove_repeated(inds):
```

Comment:
The one place where I found such a doctest was in a function that itself is internal.
That's legitimate, so the doctest should stay.
If the doctest fails because it's now emitting the deprecation warning, the output should be added to the doctest (because even for a deprecated function, we want the doctests to stay valid until the function is actually removed).